src/agent/job.ts

```diff
@@ -113,7 +113,8 @@ export class JobRunner {
                     // TODO: replace build with sender id once confirm how sent
```

Comment:
the job module is getting reworked completely so we shouldn't take changes.  
https://github.com/microsoft/steprunner